packages/manager/apps/okms/src/common/utils/location/continents.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { describe, expect, it } from 'vitest';
+
+import {
+  LOCATION_AF_NORTH_RBA,
+  LOCATION_AP_SOUTHEAST_SGP,
+  LOCATION_CA_EAST_BHS,
+  LOCATION_EU_WEST_GRA,
+  LOCATION_EU_WEST_PAR,
+  LOCATION_US_WEST_LAX,
+} from '@/common/mocks/locations/locations.mock';
+import { Location } from '@/common/types/location.type';
+
+import { getContinentCodeFromGeographyCode, groupLocationsByContinent } from './continents';
+
+describe('getContinentCodeFromGeographyCode', () => {
+  it('should return EUROPE for eu geography code', () => {
+    expect(getContinentCodeFromGeographyCode('eu')).toBe('EUROPE');
+  });
+
+  it('should return NORTH_AMERICA for ca geography code', () => {
+    expect(getContinentCodeFromGeographyCode('ca')).toBe('NORTH_AMERICA');
+  });
+
+  it('should return NORTH_AMERICA for us geography code', () => {
+    expect(getContinentCodeFromGeographyCode('us')).toBe('NORTH_AMERICA');
+  });
+
+  it('should return ASIA for ap geography code', () => {
+    expect(getContinentCodeFromGeographyCode('ap')).toBe('ASIA');
+  });
+
+  it('should return AFRICA for af geography code', () => {
+    expect(getContinentCodeFromGeographyCode('af')).toBe('AFRICA');
+  });
+
+  it('should return OTHERS for unknown geography code', () => {
+    expect(getContinentCodeFromGeographyCode('unknown' as string)).toBe('OTHERS');
+  });
+
+  it('should return OTHERS for empty string geography code', () => {
+    expect(getContinentCodeFromGeographyCode('' as string)).toBe('OTHERS');
+  });
+});
+
+describe('groupLocationsByContinent', () => {
+  const UNKNOWN_LOCATION_MOCK: Location = {
+    ...LOCATION_EU_WEST_PAR,
+    code: 'unknown',
+    geographyCode: 'unknown',
+  };
+
+  it('should return empty objects for empty locations array', () => {
+    const result = groupLocationsByContinent([]);
+    expect(result.locationsByContinent).toEqual({});
+    expect(result.continents).toEqual([]);
+  });
+
+  it('should group multiple locations from the same continent', () => {
+    const result = groupLocationsByContinent([LOCATION_EU_WEST_PAR, LOCATION_EU_WEST_GRA]);
+
+    expect(result.locationsByContinent).toEqual({
+      EUROPE: [LOCATION_EU_WEST_PAR, LOCATION_EU_WEST_GRA],
+    });
+    expect(result.continents).toEqual(['EUROPE']);
```

Comment:
```suggestion
    expect(result.continents).toEqual([continentCodes.EUROPE]);
```
you can use `continentCodes` in this test file as well